infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -246,18 +246,18 @@ public void reset() {
         String d = serverContext.getDataStore().getLogDir();
         streamLog.close();
         if (d != null) {
-            Path dir = FileSystems.getDefault().getPath(d);
-            String prefixes[] = new String[]{"log"};
-
-            for (String pfx : prefixes) {
-                try (DirectoryStream<Path> stream =
-                             Files.newDirectoryStream(dir, pfx + "*")) {
-                    for (Path entry : stream) {
+            Path dir = FileSystems.getDefault().getPath(d + File.separator + "log");
+            try (DirectoryStream<Path> stream =
```

Comment:
Actually, the reverse is true, if I recall correctly.  Before f087f5c, the LogUnit's data files weren't being stored in the `log` subdirectory.  Now they are stored in `log`.